pontoon/base/migrations/0098_move_ms_locales_to_db.py

```diff
@@ -179,7 +179,7 @@ def migrate_locales(apps, schema_editor):
         if pontoon_code in locale_map:
             locale_map[pontoon_code].ms_terminology_code = ms_code
 
-    bulk_update(locale_map.values(), update_fields=['ms_translator_code', 'ms_terminology_code'])
+    bulk_update(list(locale_map.values()), update_fields=['ms_translator_code', 'ms_terminology_code'])
```

Comment:
`bulk_update` function can't handle an instance of `dict_values`, that's why it has to be cast to a list.
```
_________________________________________________________________________________________________________ ERROR at setup of FTLTests.test_save_basic __________________________________________________________________________________________________________

request = <SubRequest '_django_setup_unittest' for <TestCaseFunction 'test_save_basic'>>, django_db_blocker = <pytest_django.plugin._DatabaseBlocker object at 0x7fa4b0469a90>

    @pytest.fixture(autouse=True, scope='class')
    def _django_setup_unittest(request, django_db_blocker):
        """Setup a django unittest, internal to pytest-django."""
        if django_settings_is_configured() and is_django_unittest(request):
            getfixturevalue(request, 'django_test_environment')
>           getfixturevalue(request, 'django_db_setup')

/usr/local/lib/python3.7/site-packages/pytest_django/plugin.py:384: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/local/lib/python3.7/site-packages/pytest_django/pytest_compat.py:3: in getfixturevalue
    return request.getfixturevalue(value)
/usr/local/lib/python3.7/site-packages/pytest_django/fixtures.py:97: in django_db_setup
    **setup_databases_args
/usr/local/lib/python3.7/site-packages/django/test/utils.py:187: in setup_databases
    serialize=connection.settings_dict.get('TEST', {}).get('SERIALIZE', True),
/usr/local/lib/python3.7/site-packages/django/db/backends/base/creation.py:69: in create_test_db
    run_syncdb=True,
/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py:131: in call_command
    return command.execute(*args, **defaults)
/usr/local/lib/python3.7/site-packages/django/core/management/base.py:330: in execute
    output = self.handle(*args, **options)
/usr/local/lib/python3.7/site-packages/django/core/management/commands/migrate.py:204: in handle
    fake_initial=fake_initial,
/usr/local/lib/python3.7/site-packages/django/db/migrations/executor.py:115: in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
/usr/local/lib/python3.7/site-packages/django/db/migrations/executor.py:145: in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
/usr/local/lib/python3.7/site-packages/django/db/migrations/executor.py:244: in apply_migration
    state = migration.apply(state, schema_editor)
/usr/local/lib/python3.7/site-packages/django/db/migrations/migration.py:129: in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
/usr/local/lib/python3.7/site-packages/django/db/migrations/operations/special.py:193: in database_forwards
    self.code(from_state.apps, schema_editor)
pontoon/base/migrations/0098_move_ms_locales_to_db.py:182: in migrate_locales
    bulk_update(locale_map.values(), update_fields=['ms_translator_code', 'ms_terminology_code'])
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

objs = dict_values([<Locale: Locale object>, <Locale: Locale object>, <Locale: Locale object>, <Locale: Locale object>, <Loca...e: Locale object>, <Locale: Locale object>, <Locale: Locale object>, <Locale: Locale object>, <Locale: Locale object>])
meta = None, update_fields = ['ms_translator_code', 'ms_terminology_code'], exclude_fields = None, using = 'default', batch_size = 110, pk_field = 'pk'

    def bulk_update(objs, meta=None, update_fields=None, exclude_fields=None,
                    using='default', batch_size=None, pk_field='pk'):
        assert batch_size is None or batch_size > 0
    
        # if we have a QuerySet, avoid loading objects into memory
        if isinstance(objs, QuerySet):
            if not objs.exists():
                return
            batch_size = batch_size or objs.count()
        else:
            if not objs:
                return
            batch_size = batch_size or len(objs)
    
        connection = connections[using]
        if meta is None:
            # TODO: account for iterables
>           meta = objs[0]._meta
E           TypeError: 'dict_values' object is not subscriptable

/usr/local/lib/python3.7/site-packages/bulk_update/helper.py:64: TypeError
```